index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <h1 class='header'>Hello World, we are software developers!</h1>
+    <section id="welcome-section" class="welcome-section">
+      <h1>Welcome Note</h1>
+        <p>This is to welcome you to our portfolio. Our names: Tinashe (Zimbabwe) and Frederick (Ghana). Be blessed!</p>
+      <a id="profile-link" href="https://github.com/Draxeytina/FrederickandTinashe.git" rel="stylesheet"></a></section>
+
+    </section>
+    <section id="projects">
+      <p class="project-tile">
+        <a href="https://github.com/Draxeytina/FrederickandTinashe.git" id="project-file">Check out our project</a>
+        <nav id="navbar" class="navbar">
+          <a href="fcc-test-suite-template-master/">Project</a>
+        </nav>
+        
+      </p>
+    </section>
+    <!--
+    Hello Camper!
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+    - The freeCodeCamp Team
+    -->
+    <script  src="js/index.js"></script>
```

Comment:
For optimisation, try running scripts in the beginning of your code 👍

Reply:
May you kindly use two spaces for indentation. This would ensure the readability of your code.